spiketoolkit/sorters/kilosort/kilosort.py

```diff
@@ -198,3 +196,7 @@ def _run(self, recording, output_folder):
         # retcode = _run_command_and_print_output_split(cmd_list)
         _call_command_split(cmd_list)
 
+    def _get_one_result(self, recording, output_folder):
+        # overwrite the SorterBase.get_result
+        sorting = se.KiloSortSortingExtractor(output_folder)
+        return sorting
```

Comment:
Normally this _get_one_reult fit the generic BaseSorter._get_one_result and should be overwritten.
But it does not matter.